scripts/common.bash

```diff
@@ -20,3 +20,14 @@ normal='\e[0m'
 bold='\e[1m'
 underline='\e[4m'
 blinking='\e[5m'
+
+# Store error code internally.
+# On the first instance of error code other than 0
+# the commit fails.
```

Comment:
Please change commit, to 'execution', since you factored it out, it is a bit misleading.